src/Athena/Views/Home/Home.cshtml

```diff
@@ -5,14 +5,34 @@
 
 @if (!User.Identity.IsAuthenticated)
 {
-    <div class="notLoggedIn" style="padding-top:5em">
-        <h1>
-            Welcome to the Athena Scheduling Assistant!
-        </h1>
-        <p>
-            This tool was designed by students for students, to allow for a more streamlined approach to allow students to schedule their course work for each semester and allow them to track thier progression through their degree.
-        </p>
+<div class="notLoggedIn" style="padding-top:5em">
+    <h1>
+        Welcome to the Athena Scheduling Assistant!
+    </h1>
+    <p>
```

Comment:
Let's make this more readable on larger screens by adding `class="flow-text"`

Reply:
Lets also give this `<p>` a `margin-bottom` of `5rem`

Reply:
thats fixed.